### DISCUSSION TITLE
How to configure proxy of Bambu RTSP stream?

### DISCUSSION THREAD

**Discussion by AdrianGarside** [bluenviron/mediamtx#2855] [Questions]
### Question

The rtsp feed is:
rtsps://bblp:[ACCESS_CODE]@10.10.10.95:322/streaming/live/1

And in the webrtc home assistant integration I need to specify it as:
rtspx://bblp:[ACCESS_CODE]@10.10.10.95:322/streaming/live/1
in case that matters.

I want to use mediamtx to proxy that stream to multiple clients to see if that allows the stream to more reliably work if there's only a single and continuous reader. That initial reader seems to largely keep working. Early on I can successfully connect multiple readers but later new connection attempts fail but eixisting ones continue to work. Hence the proxy idea.

If I specify the stream as rtsps in mediamtx yml, I get an error:
[path x1c] [RTSP source] tls: failed to verify certificate: x509: cannot validate certificate for 10.10.10.95 because it doesn't contain any IP SANs

If I try rtspx I get an unknown protocol error.

If I try rtsp:// I just get a timed out error:

**Comment by aler9** [comment]
Hello, if you look more carefully in the configuration file, you'll find a parameter called "sourceFingerprint" that allows to validate invalid TLS certificates, and also instructions on how to use it.

[a comment by an automated account is not shown]
